MNE/disp/disp.pro

```diff
@@ -107,6 +107,8 @@ SOURCES += \
     helpers/chinfomodel.cpp \
     helpers/mneoperator.cpp \
     helpers/roundededgeswidget.cpp
+#    bar.cpp \
+#    spline.cpp
```

Comment:
Please get rid of bar and spline class in the disp library, since they are now ported to dispCharts.